Sum_Of_Digits/Sum_Of_Digits.js

```diff
@@ -0,0 +1,35 @@
+/* The sum of digits of a number is the mathematical addition
+of the digits extracted from the number.
+Example: number = 12345, sum = 1 + 2 + 3 + 4 + 5 = 15
+number = 1005, sum = 1 + 0 + 0 + 5 = 6 */
+
+const sumOfDigits = (n) => {
+	let sum = 0 // initialise sum variable to 0
+	while(n > 0){
+		const b = n % 10
```

Comment:
It looks like 8 space indentation is there. Please maintain 4 space indentation at all places. The code logic looks fine.

Reply:
@singh-shreya6 done. please do review it. sorry for the wrong indentation.